WooCommerce/Classes/ViewRelated/Orders/OrderListViewModel.swift

```diff
@@ -43,6 +43,23 @@ final class OrderListViewModel {
     ///
     private var isAppActive: Bool = true
 
+    /// Used for looking up the `OrderStatus` to show in the `OrderTableViewCell`.
+    ///
+    /// The `OrderStatus` data is fetched from the API by `OrdersTabbedViewModel`.
+    ///
+    private lazy var statusResultsController: ResultsController<StorageOrderStatus> = {
+        let storageManager = ServiceLocator.storageManager
+        let descriptor = NSSortDescriptor(key: "slug", ascending: true)
```

Comment:
ℹ️ Would you mind adding this comment?

```swift
// TODO We should also include a siteID predicate here. 
```

Reply:
I saw that we were adding a predicate later in `configureStatusResultsController` and moved it here as we already know `siteID`.

Reply:
Ah, gotcha. I didn't notice that. Thanks for moving it!